#goal definition
bool saveImgsForViz
string mapName
bool start
---
#result definition
bool success
---
#feedback
